spi-api/src/main/java/de/adorsys/aspsp/xs2a/spi/domain/account/SpiAccountType.java

```diff
@@ -17,5 +17,9 @@
 package de.adorsys.aspsp.xs2a.spi.domain.account;
 
 public enum SpiAccountType {
+	// TODO a more representative list
```

Comment:
Yes, we already have a definition for this enum and we will implement it in the next version of our project. Thank you

Reply:
@jfractalus, @DG0lden  Thanks, btw. if any of you or your Nürnberg colleagues should come to[DWX 2018](https://www.developer-week.de/) in the last week of June, I'd be happy to meet you there. Anyone who doesn't have a ticket yet but plans to come, I can provide a discount code as speaker, too. I am talking about the somewhat related Java EE [Security API](https://github.com/eclipse-ee4j/security-api). Send me a mail if you want the coupon.

Reply:
@keilw Thanks!